.env

```diff
@@ -28,10 +31,8 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
 #
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
-# DATABASE_URL="postgres://symfony:ChangeMe@127.0.0.1:63494/app?sslmode=disable&charset=utf8"
-DATABASE_URL="postgresql://127.0.0.1:5432/db?serverVersion=13&charset=utf8"
-
-
+# DATABASE_URL="postgres://symfony:ChangeMe@127.0.0.1:5432/app?sslmode=disable&charset=utf8"
+DATABASE_URL="postgresql://127.0.0.1:5432/db?serverVersion=14&charset=utf8"
```

Comment:
For consistency with other ITK projects we should use MariaDB as database (cf. e.g. https://github.com/itk-dev/naevnssekretariatet/blob/develop/.env#L63).